functions/venue.js

```diff
@@ -269,6 +270,91 @@ const createVenueData_v2 = (data, context) => ({
   rooms: [],
 });
 
+const createBaseUpdateVenueData = (data, updated) => {
+  if (data.subtitle) {
+    updated.config.landingPageConfig.subtitle = data.subtitle;
+  }
+
+  if (data.description) {
+    updated.config.landingPageConfig.description = data.description;
+  }
+
+  if (data.primaryColor) {
+    if (!updated.theme) {
+      updated.theme = {};
+    }
+    updated.theme.primaryColor = data.primaryColor;
+  }
+
+  if (data.logoImageUrl) {
+    if (!updated.host) {
+      updated.host = {};
+    }
+    updated.host.icon = data.logoImageUrl;
+  }
+
+  if (data.profile_questions) {
+    updated.profile_questions = data.profile_questions;
+  }
+
+  if (data.entrance) {
```

Comment:
the `updateVenue` works now. But `createVenue` still misses `.entrance` data. We need to edit that function too to be able to add `entrance` data while creating a venue.

\+ to be able to update `entrance` data in Admin UI while editing venue, you should add `editing` property to the `EntranceInput` component

```tsx
        <EntranceInput
          register={register}
          fieldName="entrance"
          editing={state.detailsPage?.venue.entrance}
        />

```

Reply:
> couldn't check because of cors issues locally

Were you running the function emulator? That's usually the problem.

Reply:
> But `createVenue` still misses .entrance data. We need to edit that function too to be able to add entrance data while creating a venue.

@sunny-viktoryia Nice catch. I didn't try creating a new venue.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1485/commits/f395d82ff53b0de5ce1c2ce5d1a151bcd16c6729 + https://github.com/sparkletown/sparkle/pull/1485/commits/f2b495fb6dada938f332df513adaaaa39dcc3830

and a bonus one for `code_of_conduct_questions` https://github.com/sparkletown/sparkle/pull/1485/commits/fc267a813da2bc1c912d640d6062d74798059100

Reply:
super awesome! 🎉

Reply:
@0xdevalias  here is my CORS issue
![image](https://user-images.githubusercontent.com/63313732/123521631-e4a5d600-d685-11eb-8096-1a2958046584.png)

not sure why it does not work for me.
I have running:
```
npm run start
npm run firebase:emulate-functions
```

Reply:
Hrmm.. are both your .env and firebase use set to the same environment?

Reply:
ohh yes, you were right. I used `sparkle-github` for functions for some reason. 🙈 
Thank you. 🙃 It works now.